Makefile

```diff
@@ -19,7 +19,7 @@ rwildcard=$(foreach d,$(wildcard $1*),$(call rwildcard,$d/,$2) $(filter $(subst
 # Recursively find all *_test.go files from cwd & reduce to unique dir names
 HAS_TESTS = $(sort $(dir $(call rwildcard,,*_test.go)))
 # Exclude anything we explicitly don't want to test for whatever reason
-EXCLUDE_TESTS = image
+EXCLUDE_TESTS = image waveshare-epd/epd2in66b
```

Comment:
Alternatively I remove the tests for now, though I found them quite useful for myself.